src/gsi/radinfo.f90

```diff
@@ -1254,6 +1254,7 @@ subroutine radinfo_write(pe_out)
 ! !USES:
 
     use mpimod, only: mype
+    use constants, only: zero,one,r10000
```

Comment:
Nitpick:  Constants `zero` and `one` are not used in subroutine `radinfo_write`.  These constants can be removed from this `use` statement.